### PR TITLE
src: build: Fix cflags formatting issue 

### PR DESCRIPTION
**Pull request by aquilamacedo**
This commit solves the issue where setting the string as cflags "-DAMD_PRIVATE_COLOR" was not functioning as expected. The code has been updated the code to properly escape the minus symbol in cflags, making it work as expected

Closes: #934

### CODE REVIEW

**Comment by aquilamacedo** [comment]
@melissawen, I made a PR related to the issue you had opened, could you review it for me?

**Comment by davidbtadokoro** [comment]
Hey @melissawen, if you want, you can leave the code review to me. However, I feel like we need your greenlight for the behavior, as you opened the issue and surely knows much more about the domain.

**Comment by aquilamacedo** [comment]
@davidbtadokoro and @rodrigosiqueira, if you have any comments, your feedback would be more than welcome too. :)

**Review by davidbtadokoro — CHANGES_REQUESTED** [review]
Hi @aquilamacedo, and thanks for tackling this issue.

I have no inline comments. However, you claim to be escaping the minus `-` symbol, but it seems to me that what you are actually doing is surrounding the `cflags` value with double-quotes (e.g., `KCFLAGS="-DAMD_PRIVATE_COLOR"`). In case in not misunderstanding things, it may be best to change the commit message to express this.

Other than that, I tried reproducing the bug reported in #934, and this PR seems to fix it 👍

In any case, I feel that the greenlight of @melissawen is a must.

**Comment by aquilamacedo** [comment]
Hi @davidbtadokoro. Thx for your review :). I've updated the commit message to provide a more accurate description of the changes. Could you please verify if everything is in order now?

**Review by davidbtadokoro — COMMENTED** [review]
Hi @aquilamacedo! Read you updated commit message and it looks great to me 👍 

As I tested the behaviour in the last review and the code looks good, lets just wait for Melissa's greenlight to merged it 😬

**Comment by melissawen** [comment]
Just tested it. All good! Thank you guys for fixing the issue and reviewing.

**Comment by davidbtadokoro** [comment]
Hi @melissawen and thanks for testing it! @aquilamacedo, since everything seems OK, I merged your PR into the unstable. Thanks 👍

**Pull request closed without merging**
